### PR TITLE
Create Interfaces

### LINKED ISSUE #47: Create Interfaces
There are Repositories interfaces and Services interfaces in Interfaces

### PR DESCRIPTION
**Pull request by diyorzz**

### CODE REVIEW

File: StaySphere.Api/StaySphere.Domain/Enterfaces/Repositories/IRepositoryBase.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace StaySphere.Domain.Enterfaces.Repositories
+{
+    public interface IRepositoryBase<T> where T : EntityBase
+    {
+        IEnumerable<T> GetAll();
+        T FindById(int id);
+        T Create(T entity);
+        void Update(T entity);
+        void Delete(T entity);
```
Comment (SharifovDeveloper) [inline StaySphere.Api/StaySphere.Domain/Enterfaces/Repositories/IRepositoryBase.cs]: Async qiling bulani

File: StaySphere.Api/StaySphere.Domain/Enterfaces/Services/IBookingService.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+namespace StaySphere.Domain.Enterfaces.Services
+{
+    public interface IBookingService
+    {
+        BookingDto GetBooking();
+        BookingDto? GetBookingById(int id);
+        BookingDto CreateBooking(BookingForCreateDto bookingForCreateDto);
+        void UpdateBooking(BookingForUpdateDto bookingForUpdateDto);
+        void DeleteBooking(int id); 
+    }
```
Comment (SharifovDeveloper) [inline StaySphere.Api/StaySphere.Domain/Enterfaces/Services/IBookingService.cs]: Hammasini asnyc qilib chiqing

**Review by Nabiddinov — DISMISSED** [review]
good

**Review by SharifovDeveloper — APPROVED** [review]
Looks good

File: StaySphere.Api/StaySphere.Domain/Enterfaces/Services/IBookingService.cs (hunk 2 above)
Comment (diyorzz) [reply StaySphere.Api/StaySphere.Domain/Enterfaces/Services/IBookingService.cs]: OK!

File: StaySphere.Api/StaySphere.Domain/Enterfaces/Repositories/IRepositoryBase.cs (hunk 1 above)
Comment (diyorzz) [reply StaySphere.Api/StaySphere.Domain/Enterfaces/Repositories/IRepositoryBase.cs]: ok!

**Review by Nabiddinov — APPROVED** [review]
good

**Pull request merged**